client/src/app/app.component.scss

```diff
@@ -22,8 +22,12 @@
 
 
 .main-content {
-  margin: 24px;
+  margin: auto;
+  padding: 24px;
   @media (max-width: 599px) {
-    margin: 16px;
+    padding: 16px;
   }
+
+  // adds padding to the main content container on larger screens (above 1280px wide)
+  max-width: 1280px;
```

Comment:
Is 1280 the usual standard? It looks okay, but I do wonder if there are use cases where we might want to have content for the full width. (Probably not this one, but something more information intensive maybe.)

Reply:
It's one of the Angular Material breakpoints but which one is somewhat arbitrary. I suppose I could do something that better emulates what was behing done before and do a `max-width` of 80% on > small screens.

Reply:
As long as I don't get wide margins when I have my window adjusted smaller just because my physical screen is large, I can live with that. I dislike when I am forced to have large buffers at the edge on ways that prohibit me from making my own choices about how to arrange my windows. 🙂